src/App/controllers/auth.js

```diff
@@ -0,0 +1,75 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv'
+import successRes from '../utils/succHandler';
```

Comment:
You need to combine `SuccessRes` and `ErrorResponse` in a single class for responding Please!

Make sure to combine some other related functions.